src/main/java/org/web3j/console/WalletFunder.java

```diff
@@ -0,0 +1,174 @@
+                                                    + " ] "
+                                                    + msg;
+                                    System.out.write(data.getBytes());
+                                    Thread.sleep(500);
+                                }
+                                System.out.write("\n".getBytes());
+                            } catch (IOException | InterruptedException e) {
+                                e.printStackTrace();
+                            }
+                        });
+        th.start();
+    }
+
+    private static String fundWallet(String walletAddress, String token) throws Exception {
+        OkHttpClient client = new OkHttpClient();
+        ObjectMapper mapper = new ObjectMapper();
+        System.out.println("Sending funding request...");
+        Request sendEtherRequest;
+
+        if (token != null) {
+            RequestBody fundingBody =
+                    new MultipartBody.Builder()
+                            .setType(MultipartBody.FORM)
+                            .addFormDataPart("address", walletAddress)
+                            .build();
+
+            sendEtherRequest =
+                    new okhttp3.Request.Builder()
+                            .url(BASE_URL + "/send/" + token)
+                            .post(fundingBody)
+                            .build();
+        } else {
+            Request getSeedRequest =
+                    new okhttp3.Request.Builder().url(BASE_URL + "/seed/0.2").get().build();
+            Response configRawResponse = client.newCall(getSeedRequest).execute();
+
+            if (configRawResponse.code() != 200) {
+                exitError("An HTTP request failed with code: " + configRawResponse.code());
+            }
+
+            String configResponse = configRawResponse.body().string();
+
+            WalletFunderConfig config = mapper.readValue(configResponse, WalletFunderConfig.class);
+
+            AtomicBoolean found = new AtomicBoolean(false);
+            AtomicInteger intResult = new AtomicInteger(0);
+            loading("Performing proof of work to validate your request");
+
+            IntStream.range(0, Integer.MAX_VALUE)
+                    .parallel()
+                    .forEach(
+                            i -> {
+                                if (found.get()) return;
+                                String potentialHash =
+                                        Numeric.toHexString(
+                                                        sha256(
+                                                                (i + config.seed)
+                                                                        .getBytes(
+                                                                                StandardCharsets
+                                                                                        .UTF_8)))
+                                                .substring(2);
```

Comment:
Is this the standard way spotless formats this or is it a bug?

Reply:
I was about to ask..